introduction.md

```diff
@@ -40,8 +40,16 @@ ecosystem and tooling).
 
 The rest of this section describes the design and motivation around the TimescaleDB
 architecture, including why time-series data is different, and how we leverage
-its characteristics when building TimescaleDB.
+its characteristics when building TimescaleDB. 
+
+>:TIP: For a brief overview, download our new users guide [here][starter-guide].
```

Comment:
I don't know that making this a tip helps emphasize it.  It seems to be _less_ prominent.  Maybe you can make this it's own separate section with a header?

Reply:
See now?